sdks/python/apache_beam/typehints/decorators.py

```diff
@@ -378,6 +378,43 @@ def has_simple_output_type(self):
         self.output_types and len(self.output_types[0]) == 1 and
         not self.output_types[1])
 
+  def strip_pcoll_input(self):
+    # type: () -> IOTypeHints
+
+    input_type = self.input_types[0][0]
+    if isinstance(input_type, typehints.AnyTypeConstraint):
+      return self
+
+    try:
+      input_type = input_type.__args__[0]
+    except:
```

Comment:
As a general rule, don't catch all exceptions but only the ones you expect to be raised.